package.sh

```diff
@@ -0,0 +1,242 @@
+#!/bin/bash
+
+###########################################################################
+# Packaging script which creates debian and RPM packages. It optionally
+# tags the repo with the given version. 'fpm' must be on the path.
+
+INSTALL_ROOT_DIR=/opt/influxdb
+CONFIG_ROOT_DIR=/etc/opt/influxdb
```

Comment:
Technically yes, according to the Filesystem Hierarchy standard: http://refspecs.linuxfoundation.org/FHS_2.3/fhs-2.3.pdf

See section 3.7.4